package.json

```diff
@@ -55,7 +55,7 @@
     "istanbul-lib-report": "^3.0.0",
     "istanbul-reports": "^3.0.0",
     "jest": "workspace:packages/jest",
-    "jest-environment-jsdom-fifteen": "^1.0.2",
+    "jest-environment-node": "^26.5.2",
```

Comment:
this exists in the repo, I don't think we need this? _If_ we need it, it should be

```suggestion
    "jest-environment-node": "workspace:packages/jest-environment-node",
```

Reply:
We need it, otherwise the tests will fail on Node 10.x.

Reply:
really? That's so weird - what's the error?

Reply:
Yeah I will use `workspace:packages/jest-environment-node`.

This is the error `ReferenceError: globalThis is not defined`

Reply:
for `node`? really? I get that you got that error using JSDOM since typescript does something weird with `globalThis` which confuses JSDOM (see linked issue), but that's not the case for the node issue.

Reply:
@SimenB Sorry I think I've confused you. I've got that error when I don't use any `testEnvironment` in the `jest.config.ts.test.ts`. But if I use `jest-environment-node` it's all good.

Reply:
Right! But _now_ my question is if we need the `dependency` in the root `package.json` at all or if we can remove it?

Reply:
@SimenB Oh that's what you meant. I've just tried and it's working well without the root dependency. 👍